application/index.tsx

```diff
@@ -16,6 +16,7 @@ import { XMLRendererSampleData } from "../src/templates/XmlRenderer/sample";
 import { SimpleCooSampleV2 } from "../src/templates/SimpleCoo/sampleV2";
 import { SimpleCooSampleV3 } from "../src/templates/SimpleCoo/sampleV3";
 import { CertificateOfNonManipulationSampleV2 } from "../src/templates/CertificateOfNonManipulation/sampleV2";
+import { BrochureSample } from "../src/templates/Brochure/sample";
 import { App } from "./app";
```

Comment:
just curious, why does this brochure has to sit in this repo?

Reply:
oh not necessary for it to be in this repo but since it is for tradetrust I figured it was the most appropriate. Is there another renderer where we could add this in?

Reply:
think can put in oa org? spin a fresh one, do what's needed, then when no longer relevant, we can public archive it easily

Reply:
I think this might also be used as an additional gallery sample so better to leave it here, we will be pushing the corresponding oa doc to the gallery repo as a sample v4 document

Reply:
Is this template temporary only? Would it be better to fork this repo onto OA and add this temporary template to the forked repo, and then take it down when it's no longer in use?

Reply:
since we sort of know the end goal for the brochure, it kinda makes sense that it lives in a standalone repo that can be easily archived once the event has passed.

Reply:
just checked with Barry and we will be adding it to the oa gallery as a sample v4 doc

Reply:
i've sent an invite, could you join the maintainer's list?